public/js/test/cypress/commands/debugger.js

```diff
@@ -100,6 +100,7 @@ function selectScope(index) {
 }
 
 function resume() {
+  cy.wait(200);
```

Comment:
clicking resume is flakey because it can quickly toggle states. this is a hack for now, that i will investigate later.